frontend/src/components/MeetingRoom/ReportIssueButton/ReportIssueButton.tsx

```diff
@@ -21,15 +22,18 @@ const ReportIssueButton = ({ handleClick, isOpen }: ReportIssueButtonProps): Rea
   const isSmallViewport = useIsSmallViewport();
   return (
     <div className="pr-3">
-      <Tooltip title="Report issue" aria-label="open report issue menu">
+      <Tooltip
+        title={isOpen ? 'Close report issue form' : 'Open report issue form'}
+        aria-label="toggle report issue form"
+      >
         <ToolbarButton
           data-testid="report-issue-button"
           sx={{
             marginTop: '0px',
             marginRight: '0px',
           }}
           onClick={handleClick}
-          icon={<FeedbackIcon style={{ color: `${!isOpen ? 'white' : 'rgb(138, 180, 248)'}` }} />}
+          icon={<FeedbackIcon sx={{ color: isOpen ? blue.A100 : 'white' }} />}
```

Comment:
for some context this changes CSS to be consistent with the button next to it, `ParticipantListButton`